common/src/main/java/me/shedaniel/architectury/event/events/BlockEvent.java

```diff
@@ -32,32 +32,59 @@
 import org.jetbrains.annotations.Nullable;
 
 public interface BlockEvent {
-    
-    // Block interaction events
     /**
-     * Called when a player breaks a block.
```

Comment:
IMO this should still be here.

Reply:
Yeah bit is in the interface. When possible I used the javadoc over the field to point to the interface javadoc, where everything is explained